build.sbt

```diff
@@ -188,21 +201,24 @@ lazy val testsOutput = projectMatrix
     coverageEnabled := false
   )
   .defaultAxes(VirtualAxis.jvm)
-  .jvmPlatform(testTargetScalaVersions)
+  .jvmPlatform(buildScalaVersions :+ scala3, Seq(), p => p)
   .disablePlugins(ScalafixPlugin)
 
 lazy val testkit = projectMatrix
   .in(file("scalafix-testkit"))
   .settings(
     moduleName := "scalafix-testkit",
     isFullCrossVersion,
-    libraryDependencies ++= Seq(
-      googleDiff,
-      scalatest
-    )
+    libraryDependencies += googleDiff,
+    libraryDependencies ++= {
+      if (!isScala3.value)
+        Seq(scalatest)
+      else
+        Seq("org.scalatest" %% "scalatest" % "3.2.13")
```

Comment:
a different version of scalatest has to be provided, otherwise the dependency can't be found:
- `sbt.librarymanagement.ResolveException: Error downloading org.scalatest:scalatest_3:3.0.8`

I'm not sure if we can use the same version for every scala version

Reply:
For now, let's pick the latest scalatest for scala3 (can you please track it as `scalatest3V` in `Dependencies.scala` ?) and keep the old version for scala 2.x (we can align them in the next breaking release if needed).